app/src/main/java/com/example/cleanarchtest/di/ApolloNetworkModule.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.cleanarchtest.di
+
+import com.apollographql.apollo3.ApolloClient
+import com.example.cleanarchtest.util.Constants
+import org.koin.dsl.module
+
+val ApolloNetworkModule = module {
+    single { provideApolloClient() }
+}
+
+fun provideApolloClient(): ApolloClient {
+    return ApolloClient.Builder()
+        .serverUrl(Constants.GRAPH_QL_BASE_URL)
+        .build()
+}
```

Comment:
We don't need a separate file for this. The client can be provided inside `NetworkModule` only.